learning/index.mdx

```diff
@@ -8,3 +8,5 @@ description: This page is for open-source previews only and should not be synced
 This page is for open-source previews only.
 
 If you don't see your course in the following list, edit `learning/index.mdx` and restart the preview.
+
+- [Quantum computing in practice](/learning/courses/quantum-computing-in-practice)
```

Comment:
PR preview looks great! I think you're already aware, but in case not: the home page needs to be updated in closed source to link to `/learning`, not `/learning/courses`